docs/implement/security.md

```diff
@@ -0,0 +1,90 @@
+# Security in uPortal
+
+## Table of Contents
+
+1. [CORS Filter](#cors-Filter)
+
+## CORS Filter
+
+The CORS Filter is an implementation of W3C's CORS 
+(Cross-Origin Resource Sharing) specification, which
+enables cross-origin requests.
+
+### CORS Filter Configuration Options
+
+<table>
```

Comment:
Consider [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) for a more semantic and less presentation-oriented way to model this documentation. May have better accessibility properties in making more of the semantics of the document available to the browser.

Reply:
Hmm, I agree!

Reply:
Wish we were using AsciiDoc!